src/index.html

```diff
@@ -11,6 +11,18 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input data-qa="keypress" class="search-form__input search-form__input_big"
+          type="text" placeholder="Try &quot;Los Angeles&quot;">
+      </label>
```

Comment:
Need to fix code style
```suggestion
    <form
      class="search-form search-form_big"
      data-qa="big"
    >
      <label class="search-form__label search-form__label_big">
        <input
          data-qa="keypress"
          class="search-form__input search-form__input_big"
          type="text" 
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
```